van-gogh-online-webstore/src/pages/ProductsPage/index.jsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+import './style.css';
+import Navbar from '../../components/Navbar';
+import { Typography } from '@mui/material';
+import Filters from '../../components/Filters';
+
+export default function ProductsPage() {
+	return (
+        <>
+            <Navbar bgColor='white'/>
```

Comment:
Talvez seria legal fazer o filter acompanhar a barra de rolagem (nem sei se da pra fazer isso),porque se tiver muitos produtos ou a tela for muito estreita,o usuario vai ter q ficar subindo pra filtrar e depois voltar

Reply:
Então, esse normalmente não é o comportamento que eu vejo muito nos e-commerces por aí, mas se fossemos fazer algo no estilo eu acho que seria mais interessante criar paginação, aí de qualquer forma o usuário não desceria muito.